articles/azure/azs-how-config-vm-update-management.md

```diff
@@ -0,0 +1,202 @@
+---
+title: How to configure VM updates and management on Azure Stack | UKCloud Ltd
+description: Describes how use Azure Monitor to configure VM updates and management on Azure Stack
+services:
+author: Daniel Brennand
+reviewer: 
+lastreviewed: 
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Configure VM updates and management
+toc_fullpath: Users/How To/azs-how-config-vm-update-management.md
+toc_mdlink: azs-how-config-vm-update-management.md
+---
+
+# How to configure VM updates and management on Azure Stack | UKCloud Ltd
+
+## Overview
+
+Azure Monitor is the platform service that provides a single source for monitoring Azure resources. With Azure Monitor, you can visualize, query, route, archive, and otherwise take action on the metrics and logs coming from resources in Azure.
+
+This article will enable the following solutions for Azure Stack VMs:
+
+- [Azure Monitor for VMs](https://docs.microsoft.com/en-us/azure/azure-monitor/insights/vminsights-overview)
+
+- [Inventory](https://docs.microsoft.com/en-us/azure/automation/automation-vm-inventory)
+
+- [Change Tracking](https://docs.microsoft.com/en-us/azure/automation/change-tracking)
+
+- [Update Management](https://docs.microsoft.com/en-us/azure/automation/automation-update-management)
+
+## Prerequisites
+
+To complete the steps in this article, you must have appropriate access to a subscription in the Azure and Azure Stack portal.
+
+## Enabling VM Update & Management
+
+> [!WARNING]
+> Before proceeding, check [here](https://docs.microsoft.com/en-gb/azure/azure-monitor/insights/vminsights-enable-overview#supported-operating-systems) that your VMs' OS version is supported.
+
+1. Log in to the public Azure portal:
+
+    <https://portal.azure.com>
```

Comment:
Can this link be fixed to be inline with our other docs? e.g. https://docs.ukcloud.com/articles/azure/azs-how-identify-global-admins.html#identifying-the-global-administrators